README.md

```
@@ -1,2 +1,35 @@
 # influxdb3-action
-A GitHub action for setting up and configuring InfluxDB3
+
+This action installs and configures [InfluxDB3 core](https://docs.influxdata.com/influxdb3/core/).
+
+# Usage
+
+See [action.yaml](action.yaml)
+
+### Setup and configure InfluxDB3
```

Markdownlint (0.37.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)